src/test/resources/augmentation-tests.adoc

```diff
@@ -214,6 +229,42 @@ type _Neo4jLocalTime {
   second: Int
 }
 
+type _Neo4jPoint {
```

Comment:
I wonder if we can minimize that.

Either by splitting it into 2 point types.
And removing / limiting a bit of the capabilities (e.g. coordinate systems) ?

/cc @johnymontana

Reply:
This looks to match the point type used in neo4j-graphql.js, which is good. I don't think we want multiple GraphQL point types, and instead we should match the Neo4j database point type.

I think the issue @jexp is getting at here is that there are many optional fields on the point type. This stems from the fact that there aren't separate database types for 2D, or 3D - instead they are the same type. So to match that behavior the GraphQL type needs to have these optional fields. One thing that could be done in the future to simplify this is to use a GraphQL union input type (coming to the GraphQL spec soon) for the mutations and filters, which would have a LatLon, xyz, etc types wrapped up in a point union input type which I think would make it more clear what the various options for specifying a point type are.

In most cases I think users are interested in latitude, longitude with the geographic CRS, which is the default.